module/Activity/src/Activity/Controller/ActivityController.php

```diff
@@ -63,13 +63,18 @@ public function createAction()
                 $data['costs'] = '-1';    // Hack. Because empty string is seen as 0
             }
 
-            $form->setData($this->getRequest()->getPost());
+            $postData = $this->getRequest()->getPost();
+            $form->setData($postData);
 
             if ($form->isValid()) {
-                $activity = $activityService->createActivity($form->getData(\Zend\Form\FormInterface::VALUES_AS_ARRAY));
+                $activity = $activityService->createActivity(
+                    $form->getData(\Zend\Form\FormInterface::VALUES_AS_ARRAY),
+                    $postData['language_dutch'],
```

Comment:
Why is the language data passed separately? Is that not already contained in `$form->getData()` ?